### PR TITLE
changed text files to file diff

### PR DESCRIPTION
**Pull request by jstucke**
- [x] added function to find changed text files to file converage plugin
- [x] start file diff button on comparison page for found files
- [x] depends on #889
- [x] redo compare bugfix
- [x] tests
- [x] GUI optimization

### CODE REVIEW

File: src/plugins/compare/file_coverage/code/file_coverage.py [hunk 1]
```diff
@@ -164,6 +171,23 @@ def _evaluate_entropy_for_list_of_uids(self, list_of_uids, new_result, firmware_
         if non_zero_file_ids:
             new_result[firmware_uid] = non_zero_file_ids
 
+    # diverging text files
```
Comment (maringuu) [inline src/plugins/compare/file_coverage/code/file_coverage.py:174]: This comment is redundant with the function name.
Maybe the function should have a docstring.

**Review by maringuu — CHANGES_REQUESTED** [review]

File: src/plugins/compare/file_coverage/code/file_coverage.py (hunk 1 above)
Comment (jstucke) [reply src/plugins/compare/file_coverage/code/file_coverage.py:174]: removed the comment and added a docstring

**Comment by maringuu** [comment]
LGTM!

**Review by maringuu — APPROVED** [review]

**Comment by jstucke** [comment]
I will merge this since the test fail in jammy doesn't seem to have anything to do with the changes

**Pull request merged**